ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbarManager.java

```diff
@@ -99,6 +99,16 @@ public void setActions(ReactToolbar view, @Nullable ReadableArray actions) {
     view.setActions(actions);
   }
 
+  @ReactProp(name = "contentInsetStart")
```

Comment:
You should use float as a property type as in JS we use all the dimensions in dp which means that it makes sense to support non integral values